test/eventing-contrib.bash

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# TO BE REMOVED
+function header_text {
+  header $1
+}
+function upstream_knative_eventing_contrib_e2e {
+  (
+  logger.info 'Running eventing-contrib tests'
+
+  local failed=0
+
+  TEST_IMAGE_TEMPLATE="registry.svc.ci.openshift.org/openshift/knative-${KNATIVE_EVENTING_CONTRIB_VERSION}:knative-eventing-test-{{.Name}}"
+
+  cd "$KNATIVE_EVENTING_CONTRIB_HOME" || return $?
+
+  # This the namespace used to install and test Knative Eventing-Contrib.
+  export TEST_EVENTING_NAMESPACE
+  TEST_EVENTING_NAMESPACE="${TEST_EVENTING_NAMESPACE:-"knative-eventing-"$(cat /dev/urandom \
+    | tr -dc 'a-z0-9' | fold -w 10 | head -n 1)}"
+
+  export KNATIVE_DEFAULT_NAMESPACE
+  KNATIVE_DEFAULT_NAMESPACE="knative-eventing"
+
+  # Config tracing config.
+  export CONFIG_TRACING_CONFIG
+  CONFIG_TRACING_CONFIG="test/config/config-tracing.yaml"
+
+  install_tracing
+
+  source "${KNATIVE_EVENTING_CONTRIB_HOME}/openshift/e2e-common.sh"
+
+  failed=0
+
+  logger.info 'Installing Strimzi'
+  (( !failed )) && install_strimzi || failed=$?
+
+  # run_e2e_tests defined in eventing-contrib
+  logger.info 'Starting eventing-contrib tests'
+  (( !failed )) && run_e2e_tests_workaround || failed=$?
+
+  print_test_result ${failed}
+
+  return $failed
+  )
+}
+
+function run_e2e_tests_workaround(){
```

Comment:
We can get rid of this workaround. This function was created because there were some broken/missing things in eventing-contrib/e2e-common.sh.

However, I would do this as a follow up. I wouldn't want to experiment if things have identical behavior when using the eventing-contrib one.

Let's leave as is.

Reply:
PR: https://github.com/matzew/serverless-operator/pull/5